packages/slice-machine/components/ChangesItems/ChangesItems.tsx

```diff
@@ -147,6 +158,12 @@ export const ChangesItems: React.FC<ChangesItemsProps> = ({
             slices={sliceFilterFn(unSyncedSlices)}
             defaultVariationSelector={defaultVariationSelector}
           />
+          <RenameSliceModal
```

Comment:
Actually, I'm pretty sure the rename and delete options shouldn't be in the slice card in the changes page